### PR TITLE
update some incorrect description

### PR DESCRIPTION
**Pull request by guangxuli**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,8 +64,19 @@ We can also [create new apps using OpenShift template files](https://docs.opensh
...
+	│       ├── nodejs.json
+	│       ├── nodejs-mongodb.json
+	│       └── nodejs-mongodb-persistent.json
+	├── package.json
+	├── README.md
+	├── server.js
+	├── tests
+	│   └── app_test.js
+	└── views
+	    └── index.html
```
Comment (bparees) [inline README.md]: this doesn't look properly edited into the existing content.

**Comment by guangxuli** [comment]
@bparees done

**Comment by Fredy89** [comment]
@guangxuli  do you can help me?

**Comment by guangxuli** [comment]
@Fredy89 what can i do for you?

**Pull request merged**